trade-finance-manager-ui/component-tests/utilisation-reports/_macros/report-reconciliation-table.component-test.js

```diff
@@ -35,6 +36,11 @@ describe(component, () => {
 
       wrapper.expectElement(`${rowSelector} th`).toHaveCount(1);
       wrapper.expectElement(`${rowSelector} th:contains("${summaryItem.bank.name}")`).toExist();
+      if (summaryItem.status === 'REPORT_NOT_RECEIVED') {
```

Comment:
As this is a `.js` file we should use `UTILISATION_REPORT_RECONCILIATION_STATUS.REPORT_NOT_RECEIVED` here.